pypesto/optimize/optimizer.py

```diff
@@ -623,3 +627,53 @@ def minimize(
 
     def is_least_squares(self):
         return False
+
+
+class ScipyDifferentialEvolutionOptimizer(Optimizer):
+    """
+    Global optimization using scipy's differential evolution optimizer.
+    Package homepage: https://docs.scipy.org/doc/scipy/reference/generated
+                        /scipy.optimize.differential_evolution.html
+    Parameters
+        ----------
+        maxiter: used to calculate the maximal number of funcion evaluations by
+                 maxfevals = (maxiter + 1) * popsize * len(x)
+        popsize: default value 15
+        len(x): length of the argument of the function to be optimized
+                in the form of a 1-D array
```

Comment:
These are not the real options, but the "options inside the options". Please only name "options" as a parameter here, and possibly mention the above arguments as examples of what can be passed to options. len(x) should not be needed I think.

Reply:
len(x) is needed for the calculation of maxfevals so I would leave it here